test/integration/migrate/test_single_per_migration_trx_disabled/down.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { isPostgreSQL } = require('../../../util/db-helpers');
+
+exports.up = async function () {};
+
+exports.down = async function (knex) {
+  if (isPostgreSQL(knex)) {
```

Comment:
this doesn't seem to be the right place for skipping the test, could you move it into a test itself? there are plenty of examples of how db-specific tests are handled

Reply:
Moved to test